### PR TITLE
Apply methods

### PR DESCRIPTION
**Pull request by laingbeth**

### CODE REVIEW

**Comment by rgpass** [comment]
Not sure why I was unassigned. :(

File: talkative_robot_beth.rb [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+ # not sure if I am on the right track; I understand that the get can be put into a method 
+ # not sure if I should be putting other things into a method.  Thanks
+
+
+puts "What is your name?"
+name = gets.chomp
+
+#method 
+def get_age
+puts "How old are you?"
```
Comment (rgpass) [inline talkative_robot_beth.rb:11]: This should be indented.

File: talkative_robot_beth.rb [hunk 2]
```diff
@@ -0,0 +1,84 @@
+ ##Lab/Homework - Talkative Robot with methods added
+ # not sure if I am on the right track; I understand that the get can be put into a method 
+ # not sure if I should be putting other things into a method.  Thanks
+
+
+puts "What is your name?"
+name = gets.chomp
```
Comment (rgpass) [inline talkative_robot_beth.rb:7]:
``` ruby
def get_name
  puts "What is your name?"
  return gets.chomp
end
name = get_name
```

File: talkative_robot_beth.rb [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+gender = get_gender
+
+puts "So, you're a #{gender}."
+
+if gender == "M"  
+	puts "Hi there, Champ" 
+else gender == "F"  
+	puts "Hello Gorgeous!" 
+end
```
Comment (rgpass) [inline talkative_robot_beth.rb:33]: Could get this down to one line via ternary.

File: talkative_robot_beth.rb [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+def fav_robot
+	puts "What would your favorite movie robot say right now?"
+	#return fav_robot
```
Comment (rgpass) [inline talkative_robot_beth.rb:37]: This is just asking without returning anything.

File: talkative_robot_beth.rb [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+end
+
+def olive_fav
+	puts "What is your favorite kind of olive, green or black?"
+	olive_fav = gets.chomp
```
Comment (rgpass) [inline talkative_robot_beth.rb:42]: This one does return it. Could change this last line to `return gets.chomp`

File: talkative_robot_beth.rb [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+if gender == "F" && age <= 10 
+	puts "You're a young girl"
+elsif gender == "M" && age <= 10
+	puts "You're a young boy"
+elsif gender == "F" && age > 120
+	puts "You're a great-great grandmother"
+elsif gender == "M" && age > 120
+	puts "You're a great-great grandfather"
+end
```
Comment (rgpass) [inline talkative_robot_beth.rb:73]: Could clean these conditionals up with three boolean methods of `male?(gender)`, `young?(age)`, and `old?(age)`

**Comment by rgpass** [comment]
Since this is closed and merged, put these changes in your Collections lab and I'll review it there.

**Comment by laingbeth** [comment]
Thanks GB,
I tried to merge out because I had some wrong file issues that I think I have fixed, sorry about the thrash.
Beth

**Pull request merged**